README.md

```diff
@@ -112,6 +112,9 @@ Create a new WebView window. Calling this function will block application execut
 
 These functions below must be invoked after webview windows is created with `create_window()`. Otherwise an exception is thrown.
 
+- `webview.set_title(url)`
```

Comment:
Hey @r0x0r is it really `url` as parameter? Shouldn't it be a text or string?

Reply:
Changed